tests/Functional/src/com/microsoft/aad/adal/test/DefaultTokenCacheStoreTests.java

```diff
@@ -74,7 +74,7 @@ protected void tearDown() throws Exception {
     }
 
     public void testSharedCache() throws GeneralSecurityException, IOException {
```

Comment:
I think what this test intends to test is, if the cache is shared, then the returned client id should be as expected. It doesn't actually test cache is shared.